modules/liveIntentIdSystem.js

```diff
@@ -187,9 +195,8 @@ export const liveIntentIdSubmodule = {
       // old versions stored lipbid in unifiedId. Ensure that we can still read the data.
       const lipbid = value.nonId || value.unifiedId
       if (lipbid) {
-        value.lipbid = lipbid
-        delete value.unifiedId
-        result.lipb = value
+        result.lipb = { ...value, lipbid }
+        delete result.lipb.unifiedId
```

Comment:
It looks like this code does not delete the `unifiedId`. Is it intended? If yes, why?